Github example is failing

I tried to run github example but it failed, because the github repository `perrygoy/screenpy` no longer exits, now is `ScreenPyHQ/screenpy`

> **Note**
> The written test is finding for one result, but finding `ScreenPyHQ/screenpy` it will have multiple results and fails, so probably we want to find for an specific repo, for instance `ScreenPyHQ/screenpy_examples`

```bash
$ python -m pytest features/ --alluredir allure_report/
=================================================================================== test session starts ====================================================================================
platform linux -- Python 3.10.6, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/scot3004/proyectos/comunidades/screenpy/screenpy_examples/screenpy_selenium/github
plugins: html-3.1.1, variables-2.0.0, allure-pytest-2.13.1, metadata-1.11.0, base-url-2.0.0
collected 2 items                                                                                                                                                                          

features/test_github_search.py F.                                                                                                                                                    [100%]

========================================================================================= FAILURES =========================================================================================
_________________________________________________________________________________ test_search_for_screenpy _________________________________________________________________________________

Perry = Perry

    @act("Search")
    @scene("Search for the ScreenPy repository on GitHub")
    def test_search_for_screenpy(Perry: Actor) -> None:
        """GitHub search finds the screenpy repository."""
        given(Perry).was_able_to(Open.their_browser_on(URL))
        when(Perry).attempts_to(SearchGitHub.for_text("perrygoy/screenpy"))
>       then(Perry).should(
            See.the(SearchResultsMessage(), DoesNot(ContainTheText("couldn’t"))),
            See.the(SearchResultsMessage(), ReadsExactly("1 repository result")),
            See.the(NumberOfSearchResults(), IsEqualTo(1)),
        )

features/test_github_search.py:50: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/home/scot3004/.pyenv/versions/3.10.6/lib/python3.10/site-packages/screenpy/actor.py:165: in should
    return self.attempts_to(*actions)
/home/scot3004/.pyenv/versions/3.10.6/lib/python3.10/site-packages/screenpy/actor.py:157: in attempts_to
    self.perform(action)
/home/scot3004/.pyenv/versions/3.10.6/lib/python3.10/site-packages/screenpy/actor.py:169: in perform
    action.perform_as(self)
/home/scot3004/.pyenv/versions/3.10.6/lib/python3.10/site-packages/screenpy/pacing.py:90: in wrapper
    retval = n_func(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <screenpy.actions.see.See object at 0x7faced39aef0>, the_actor = Perry

    @beat("{} sees if {question_to_log} is {resolution_to_log}.")
    def perform_as(self: SelfSee, the_actor: Actor) -> None:
        """Direct the Actor to make an observation."""
        if isinstance(self.question, Answerable):
            value: object = self.question.answered_by(the_actor)
        else:
            # must be a value instead of a question!
            value = self.question
            aside(f"the actual value is: {value}")
    
        reason = ""
        if isinstance(self.question, ErrorKeeper):
            reason = f"{self.question.caught_exception}"
    
>       assert_that(value, self.resolution, reason)
E       AssertionError: 
E       Expected: not a string containing 'couldn’t'
E            but: but was "We couldn’t find any repositories matching 'perrygoy/screenpy'"

/home/scot3004/.pyenv/versions/3.10.6/lib/python3.10/site-packages/screenpy/actions/see.py:65: AssertionError
------------------------------------------------------------------------------------ Captured log call -------------------------------------------------------------------------------------
ERROR    screenpy:stdout_adapter.py:44     ***ERROR***

AssertionError: 
Expected: not a string containing 'couldn’t'
     but: but was "We couldn’t find any repositories matching 'perrygoy/screenpy'"
===================================================================================== warnings summary =====================================================================================
../../../../../../.pyenv/versions/3.10.6/lib/python3.10/site-packages/selenium/webdriver/remote/remote_connection.py:24
  /home/scot3004/.pyenv/versions/3.10.6/lib/python3.10/site-packages/selenium/webdriver/remote/remote_connection.py:24: DeprecationWarning: 'urllib3[secure]' extra is deprecated and will be removed in a future release of urllib3 2.x. Read more in this issue: https://github.com/urllib3/urllib3/issues/2680
    import urllib3

features/test_github_search.py: 25 warnings
  /home/scot3004/.pyenv/versions/3.10.6/lib/python3.10/site-packages/selenium/webdriver/remote/remote_connection.py:359: DeprecationWarning: HTTPResponse.getheader() is deprecated and will be removed in urllib3 v2.1.0. Instead use HTTPResponse.headers.get(name, default).
    if resp.getheader('Content-Type'):

features/test_github_search.py: 25 warnings
  /home/scot3004/.pyenv/versions/3.10.6/lib/python3.10/site-packages/selenium/webdriver/remote/remote_connection.py:360: DeprecationWarning: HTTPResponse.getheader() is deprecated and will be removed in urllib3 v2.1.0. Instead use HTTPResponse.headers.get(name, default).
    content_type = resp.getheader('Content-Type').split(';')

-- Docs: https://docs.pytest.org/en/stable/warnings.html
================================================================================= short test summary info ==================================================================================
FAILED features/test_github_search.py::test_search_for_screenpy - AssertionError:
```

--- a/screenpy_selenium/github/features/test_github_search.py
+++ b/screenpy_selenium/github/features/test_github_search.py
@@ -42,11 +42,11 @@ def fixture_actor() -> Generator:
 
 
 @act("Search")
-@scene("Search for the ScreenPy repository on GitHub")
+@scene("Search for the ScreenPy examples repository on GitHub")
 def test_search_for_screenpy(Perry: Actor) -> None:
-    """GitHub search finds the screenpy repository."""
+    """GitHub search finds the screenpy examples repository."""
     given(Perry).was_able_to(Open.their_browser_on(URL))
-    when(Perry).attempts_to(SearchGitHub.for_text("perrygoy/screenpy"))
+    when(Perry).attempts_to(SearchGitHub.for_text("ScreenPyHQ/screenpy_examples"))
     then(Perry).should(
         See.the(SearchResultsMessage(), DoesNot(ContainTheText("couldn’t"))),
         See.the(SearchResultsMessage(), ReadsExactly("1 repository result")),

